### PR TITLE
[vscode][4/n] Setup Env Variables: Add helper text to existing env path

### PR DESCRIPTION
**Pull request by rossdanlm**
[vscode][4/n] Setup Env Variables: Add helper text to existing env path

This PR I had some issues with reading the env template, so for now I'm splitting up this logic and will follow up next PR

## Test Plan

https://github.com/lastmile-ai/aiconfig/assets/151060367/6aad117f-0c55-4549-8f46-1b438c6ebf4b

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1284).
* #1285
* __->__ #1284
* #1283
* #1280

### CODE REVIEW

File: vscode-extension/src/extension.ts [hunk 1]
```diff
@@ -757,18 +757,33 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
+    context.extensionUri,
+    "static",
+    "env_template.env"
+  );
+
   if (fs.existsSync(envPath)) {
-    vscode.window.showInformationMessage(
-      "Env file already exists, will implement next PR"
-    );
+    var helperText: string = "\ntest, will change next PR";
```
Comment (Ankush-lastmile) [inline vscode-extension/src/extension.ts:767]: nit: don't use `var`, use `let`

File: vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -780,19 +795,20 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
+  if (doc) {
+    vscode.window.showTextDocument(doc, {
+      preview: false,
+      // Tried using vscode.ViewColumn.Active but that overrides existing
+      // walkthrough window
+      viewColumn: vscode.ViewColumn.Beside,
+    });
+    vscode.window.showInformationMessage(
+      "Please define your environment variables."
+    );
```
Comment (Ankush-lastmile) [inline vscode-extension/src/extension.ts:811]: please add if/else to catch if this needed

**Review by Ankush-lastmile — APPROVED** [review]
lgtm, approving to unblock

File: vscode-extension/src/extension.ts [hunk 3]
```diff
@@ -757,18 +757,33 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
+    "static",
+    "env_template.env"
+  );
+
   if (fs.existsSync(envPath)) {
-    vscode.window.showInformationMessage(
-      "Env file already exists, will implement next PR"
-    );
+    var helperText: string = "\ntest, will change next PR";
+    // fs.readFile(envTemplatePath.fsPath, function read(err, data) {
```
Comment (saqadri) [inline vscode-extension/src/extension.ts:768]: nit: Remove commented out code

**Review by saqadri — APPROVED** [review]

File: vscode-extension/src/extension.ts (hunk 3 above)
Comment (rossdanlm) [reply vscode-extension/src/extension.ts:768]: I change it in the next PR, just wasn't able to get it to work and wanted quick feedback

File: vscode-extension/src/extension.ts (hunk 2 above)
Comment (rossdanlm) [reply vscode-extension/src/extension.ts:811]: https://github.com/lastmile-ai/aiconfig/issues/1287

**Pull request merged**